requirements_versions.txt

```diff
@@ -1,32 +1,33 @@
-GitPython==3.1.32
-Pillow==9.5.0
-accelerate==0.21.0
-blendmodes==2022
-clean-fid==0.1.35
-diskcache==5.6.3
-einops==0.4.1
-facexlib==0.3.0
-fastapi==0.94.0
-gradio==3.41.2
-httpcore==0.15
-inflection==0.5.1
-jsonmerge==1.8.0
-kornia==0.6.7
-lark==1.1.2
-numpy==1.26.2
-omegaconf==2.2.3
-open-clip-torch==2.20.0
-piexif==1.1.3
-psutil==5.9.5
-pytorch_lightning==1.9.4
-resize-right==0.0.2
-safetensors==0.4.2
-scikit-image==0.21.0
-spandrel==0.1.6
-tomesd==0.1.3
-torch
-torchdiffeq==0.2.3
-torchsde==0.2.6
-transformers==4.30.2
-httpx==0.24.1
-pillow-avif-plugin==1.4.3
+GitPython==3.1.43
+Pillow==10.3.0
```

Comment:
if PIL is upgreaded to >= 10.0 then some things will have to be re-witten
for example
- https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/16558#issuecomment-2420656954

similar issues can exist in other dependencies
need lots of testing